js/components/graph/Graph.js

```diff
@@ -174,8 +178,7 @@ export default class Graph extends React.Component {
           width: data.width,
           height: data.height,
           zoomFactor: 1,
-          horizontalPanFactor: 0,
-          verticalPanFactor: 0,
+          viewBoxPos: {x: 0, y: 0},
```

Comment:
The previous code sets the viewbox width/height as a max of the container's width/height and the `data.width` and `data.height`, so I think you should do the same thing here.